src/solver/implicitgemm_util.hpp

```diff
@@ -514,6 +517,11 @@ static inline bool IsValidBlockwiseGemmXdlops(const ConvolutionContext& ctx,
                                               const int GemmNPerWave,
                                               const int GemmKPack)
 {
+#if WORKAROUND_SWDEV_251757
+    if(ctx.IsFp32() && GemmKPerBlock == 1 && GemmKPack == 8)
```

Comment:
Please use `HipCompilerVersion()` to meet https://github.com/AMDComputeLibraries/MLOpen/issues/2331. This would avoid perf drop when MIOpen is run with older ROCm that is free from SWDEV-251757.

Reply:
We didn't test the cases using older compilers, so we are not sure if older compilers works for these cases.

The compiler team is fixing the issue for future ROCm, and when it's fixed and released, we can add version detection here to enable this solver for the new compiler.

Reply:
Good, thanks.